tests/test_abstract_repr.py

```diff
@@ -278,7 +282,15 @@ def validate_schema(instance):
         encoding="utf-8",
     ) as f:
         schema = json.load(f)
-    jsonschema.validate(instance=instance, schema=schema, registry=REGISTRY)
+    if DEPRECATED_JSONSCHEMA:
+        assert RESOLVER is not None
+        jsonschema.validate(
+            instance=instance, schema=schema, resolver=RESOLVER
+        )
+    else:
+        jsonschema.validate(
```

Comment:
nit: Maybe you can add here 
```suggestion
        assert RESOLVER is None
        jsonschema.validate(
```

Reply:
Yes I hesitated to add an assert here, I will add it ;)